### ISSUE TITLE
Surface compile Errors and Warnings

### ISSUE THREAD

**Issue by nqthqn** [wking-io/elm-live#121]
... with clickable line numbers that open the file in your `EDITOR` and go to the line number :).

**Comment by architectcodes** [comment]
Oh man that would be lovely!

You might have read the [comment](https://github.com/architectcodes/elm-live/issues/75#issuecomment-402547556) I pinged you in the other issue @nqthqn – would be super happy to see someone implementing this and the elm-live project getting a life of its own with a new team of owners. I’d love to support it but I now see it will be tough moving on.

**Comment by wking-io** [comment]
I plan on looking into getting this setup sooner rather than later! 👌

**Comment by lucamug** [comment]
Doing some test about this. I tried using this url in chrome but it just open Atom without opening the file:

atom://open?url=file:///etc/profile

at the same time this url properly open the file inside the browser:

file:///etc/profile

Is that working for you?

**Comment by wking-io** [comment]
Yeah I have been looking into this one and getting it to open consistently across OS's is a tough one. Also across different editors. Because I use VSCode. I know a lot of Elm users use Atom and then there is Sublime. So this one may be tough. Do you know of any example CLI tools that do this already we can look at?

**Comment by lucamug** [comment]
So, the issue above was caused by the fact that I didn't install the package "open" (oops), I though it was installed by default.

The link works well. "vscode" should work fine too using https://github.com/shengyou/vscode-handler or something similar.

This is on Mac, I don't know with other OS.

I added this feature in this branch where I am collecting all recent enhancements: https://github.com/lucamug/elm-live/tree/enhancements

The link is only added in the browser, not in the console (it could there too but I didn't want to pollute the console). I added a new "--ide" parameter to elm-live that now default to "atom" (should probably default to false) that can be used to select the proper editor.

<img width="1440" alt="open-in-vscode" src="https://user-images.githubusercontent.com/5551094/58759513-4a3a3d80-8566-11e9-85d7-a14a775ed41b.png">
